src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5235,4 +5235,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_FilterIsConstantFalseRedundantTryCatch_Title" xml:space="preserve">
     <value>Filter expression is a constant 'false'. </value>
   </data>
+  <data name="ERR_RangeNotFound" xml:space="preserve">
+    <value>Cannot create a range value because the compiler required type '{0}' cannot be found. Are you missing a reference?</value>
```

Comment:
is "are you missing a reference" the standard way we ask that sort of question?

Reply:
I think I copied the range message from `ERR_DynamicAttributeMissing`.

```
<data name="ERR_DynamicAttributeMissing" xml:space="preserve">
  <value>Cannot define a class or member that utilizes 'dynamic' because the compiler required type '{0}' cannot be found. Are you missing a reference?</value>
</data>
```

`ERR_TupleElementNamesAttributeMissing`, `ERR_DynamicAttributeMissing`, and `ERR_DynamicRequiredTypesMissing` all have that wording.